markdown/developers/testing_with_gitpod.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Testing with Gitpod
+subtitle: Testing, Code review and Website dev.
+---
+
+# Introduction
+
+Gitpod is an open-source developer platform that can be spun up quickly from a Git repository, containing all the programs needed to run your pipelines. It has many purposes, including: testing pipelines, learning Nextflow, reviewing/editing code and live views of edited markdown/HTML for webpage development. In addition, within Gitpod, any change can be pushed to branches for review. 
+
+Gitpod is free (with a GitHub/GitLab/Bitbucket account) for use up to 50hours/month and allows 4 parellel workspaces (as of March 2022). It runs in your browser (recommended with Chrome, Firefox or Edge). You can find their extensive documentation [here](https://gitpod.io/). 
+
+Each nf-core repository (including the webpage) is integrated with Gitpod. When you click on the Gitpod button (see below), this clones a copy of the repository into your Gitpod instance with all the software necessary to run tests and display files. A terminal is also available to run test and workflow debugging.
```

Comment:
It seems this is not quite true yet, so we should add what to look for and how to deal with it if it's not there.
```suggestion
Each nf-core repository (including the nf-core webpage) should have a ready-to-go Gitpod environment waiting to be started. Since this is a new nf-core feature, check for the `.gitpod.yml` file in the root of the repository. If it's there, you're ready to start using Gitpod. If the file is absent, then make a fork of the repository, start a new branch, and then in your new branch copy the `.gitpod.yml` file from the [nf-core tools repository](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/.gitpod.yml). Then start Gitpod from this branch.

By clicking on the Gitpod button (see below), your repository is cloned into your Gitpod instance with all the software necessary to run tests and display files. A terminal is also available to run test and workflow debugging.
```